packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx

```diff
@@ -91,18 +88,19 @@ function ChainTypeInfoRow({
 
 function NetworkRow({
   chainId,
+  selectedChainId,
```

Comment:
nit: since we don't use `selectedChainId` for anything else, would it make more sense to just pass `isSelected`?